assets/css/leaderboard.css

```diff
@@ -146,3 +147,55 @@ th:not(.detail-header)  {
     background-color: #000000; /* Darker shade of blue on hover */
     transform: scale(1.05); /* Slightly increase size on hover */
 }
+
+thead {
+  position: sticky;
+  top: 0;
+  background-color: #f1f8ff; /* Background for better visibility */
+  background-clip: border-box;
+  z-index: 100; /* Ensure the header stays above other content */
+  margin: 0;
+}
+
+#leaderboard thead th{
+    z-index: 100; /* Ensure the header stays above other content */
+    /* border-bottom: 1px solid black;  */
+    background-color: #f1f8ff; /* Background for better visibility */
+}
+
+/* Sticky left columns */
+  #leaderboard tbody td:first-child,
+  #leaderboard  thead .sticky-top-column,
+  #leaderboard thead .sticky-second-column:first-child {
+    position: sticky;
+    left: 0;
+    z-index: 30;
+    background-color: #f1f8ff;
+  }
+
+  #leaderboard tbody td:nth-child(2),
+  #leaderboard thead .sticky-second-column:nth-child(2) {
+    position: sticky;
+    left: 51px;
```

Comment:
Where does the value `51px` come from?

Reply:
The 51px is the fixed column width (50px + 1px border), as the column width is constant across different screen sizes based on previous implementation.